src/NetAnalyzers/CSharp/Microsoft.NetCore.Analyzers/Performance/CSharpPreferDictionaryTryMethodsOverContainsKeyGuardFixer.cs

```diff
@@ -119,14 +116,17 @@ public override async Task RegisterCodeFixesAsync(CodeFixContext context)
                     typeSyntax = IdentifierName(Var);
                 }
 
-                var outArgument = generator.Argument(RefKind.Out,
+                var outArgument = (ArgumentSyntax)generator.Argument(RefKind.Out,
                     DeclarationExpression(
                         typeSyntax,
                         SingleVariableDesignation(Identifier(Value))
                     )
                 );
-                var tryGetValueInvocation = generator.InvocationExpression(tryGetValueAccess, keyArgument, outArgument);
-                editor.ReplaceNode(containsKeyInvocation, tryGetValueInvocation);
+                editor.ReplaceNode(
+                    containsKeyInvocation,
+                    containsKeyInvocation
```

Comment:
nit: the original form of using an extra local makes it easier to read.

Reply:
⏱️  Will amend to return to previous form

Reply:
➡️ Now fixed